plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -589,6 +591,14 @@ instance uni ~ DefaultUni => ToBuiltinMeaning uni DefaultFun where
         makeBuiltinMeaning
             (verifySignature @EvaluationResult)
             (runCostingFunThreeArguments . paramVerifySignature)
+    toBuiltinMeaning VerifyEcdsaSecp256k1Signature =
+        makeBuiltinMeaning
+            verifyEcdsaSecp256k1Signature
+            mempty
```

Comment:
We still don't have any way for contributors to add cot models, so I'll add models for these when this is merged.